build.gradle

```diff
@@ -32,32 +33,38 @@ buildscript {
 
 subprojects {
   afterEvaluate { project ->
-    task sourcesJar(type: Jar) {
-      from android.sourceSets.main.java.srcDirs
-      classifier = 'sources'
-    }
 
-    task javadoc(type: Javadoc) {
-      source = android.sourceSets.main.java.srcDirs
-      classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
-      classpath += fileTree(dir: "$buildDir/intermediates/exploded-aar/", include: "**/classes.jar")
-      classpath += configurations.compile
-      // Exclude generated files
-      exclude '**/BuildConfig.java'
-      exclude '**/R.java'
-    }
+    // These two projects are JARed and DOCed using their own build.gradle files
+    if (!project.projectDir.absolutePath.contains("backbone") && !project.projectDir.absolutePath.contains("skin")) {
```

Comment:
We deploy to bintray and ask bintray to forward on to jcenter (which is nice because jcenter is a default artifact repo for Android Studio). 

Bintray requires source jars and javadocs to enable maven magic that also helps us as devs (probably not needed now that we're using submodules) which is having access to published source code and javadocs instead of decompiled code in the IDE.